### DISCUSSION TITLE
[Request] Entity hider having an on-screen flag showing it is currently on

### DISCUSSION THREAD

**Discussion by Cinspawn** [runelite/runelite#18548] [Ideas]
I only use entity hider for certain things such as bosses, GotR and such. But I ALWAYS forget to turn it off afterwards while I do like to see players in the world. Something as simple as a "Entity hider is on" textbox above the in-game chat would be wonderful. No textbox needed when it's off.

**Comment by cdfisher** [comment]
If someone doesn't notice that players are being hidden, are they really going to notice a small overlay that basically tells them the same thing? 

Realistically, I personally don't think so.

**Reply by Cinspawn** [reply]
Oh absolutely. It's the difference between 'oh, guess there's no one around' (a potential normal occurance) and 'oh right, entity hider is on, says so right here' (not a normal occurance to see the overlay) 

The things is I do notice players are hidden, but it takes me a few hours sometimes. Or until I get jumpscared by phantom drops from other people.
